test/functionalTest/cases/0947_v2_list_all_entities/GET_v2_entities.test

```diff
@@ -277,13 +277,16 @@ Date: REGEX(.*)
 03. GET /v2/entities
```

Comment:
Not really part of this PR by a (very :) nice to have in it, in my opinion. I tend too think in /v1 and /v2 as separe blocks and I'm atticipating the situation in which /v1 would be removed (if we have mixed .test in that case, removal of /v1 would be harder).

What do you think?